subnets/config_test.go

```diff
@@ -34,8 +35,8 @@ func TestValid(t *testing.T) {
 			name: "invalid consensus parameters",
 			s: Config{
 				ConsensusParameters: snowball.Parameters{
-					K:     2,
-					Alpha: 1,
+					K:               2,
+					AlphaPreference: 1,
```

Comment:
nit: should we add a case for AlphaConfidence violation as well?

Reply:
Eh... This test is just making sure that the config verification is verify the consensus params... It could just as easily modify the beta values 🤷. I don't think it makes sense to add multiple invalid consensus params here - those are already tested in the params tests.